plonetheme/barceloneta/theme/less/header.plone.less

```diff
@@ -59,19 +59,20 @@
         max-width: 50%;
         padding-top: 0;
         position: relative;
-        width: 250px;
+        width: 260px;
```

Comment:
if we dont specify searchbox width here, this happens:
![width_remove](https://cloud.githubusercontent.com/assets/17498833/14304025/07b7e10e-fbcd-11e5-9273-168061811b95.png)

notice the misalignment of searchbox.